test/base_test_fixture.h

```diff
@@ -1012,6 +1013,24 @@ struct base_test_fixture
         REQUIRE(!connection.dbms_version().empty());
     }
 
+    void get_info_test()
+    {
+        // A generic test to exercise the DBMS info API is callable.
+        // DBMS-specific test (MySQL, SQLite, etc.) may perform extended checks.
+        nanodbc::connection connection = connect();
+        REQUIRE(!connection.get_info<nanodbc::string_type>(SQL_DRIVER_NAME).empty());
+        REQUIRE(!connection.get_info<nanodbc::string_type>(SQL_ODBC_VER).empty());
+
+       // Test SQLUSMALLINT resutls
+        REQUIRE(connection.get_info<unsigned short>(SQL_ODBC_INTERFACE_CONFORMANCE) > 0);
```

Comment:
You are right, I was trying to find a value that would return more than 0 for all drivers and somehow missed this was returning a `SQLINTEGER` rather than `SQLUSMALLINT`, will fix.